examples/minimal/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "moduleResolution": "node",
+    "esModuleInterop": true,
+    "strict": true
+  },
+  "include": ["server", "client"],
+  "exclude": ["**/node_modules"]
```

Comment:
`node_modules` are excluded by default but it's fine if you want to be explicit

Reply:
fixed